### DISCUSSION TITLE
对于qiankun使用上的疑惑

### DISCUSSION THREAD

**Discussion by CHENLiver** [umijs/qiankun#2646] [General]
## What happens?

对于qiankun的文档有些不太清晰
问题1 ：qiankun使用loadMicroApp手动加载微应用，还需要先执行registerMicroApps和start吗 （我测试过只执行loadMicroApp也能加载微应用）
问题2 ：如使用了registerMicroApps start ，不使用loadMicroApp 如何销毁微应用呢

## 相关环境信息

- **qiankun 版本** 2.10.11
- **浏览器版本**：google 115.x
- **操作系统**：mac

**Comment by Aeero** [comment]
1. 不需要
2. 我理解应用export生命周期unmount方法里销毁页面即可，实际上qiankun实例还是保留的
